app/uk/gov/hmrc/propertylinking/controllers/valuations/ValuationsController.scala

```diff
@@ -74,24 +75,44 @@ class ValuationsController @Inject()(
         } else if (link.pending && link.assessments.size == 1 && isSkipAssessment) {
           Future.successful(Redirect(getViewSummaryCall(link.uarn, link.pending, owner)))
         } else {
-          propertyLinks.clientPropertyLink(submissionId).map {
-            case None => notFound
-            case Some(clientPropertyLink) => {
-              Ok(views.html.dashboard.assessments(
-                model = AssessmentsVM(
-                  assessmentsWithLinks = link.assessments
-                    .sortBy(_.currentFromDate.getOrElse(LocalDate.of(2017, 4, 7)))(
-                      Ordering.by[LocalDate, Long](_.toEpochDay))
-                    .reverse
-                    .map(decideNextUrl(submissionId, link.authorisationId, _, link.pending, owner)),
-                  backLink = calculateBackLink(owner, Some(clientPropertyLink.client)),
-                  address = link.address,
-                  capacity = link.capacity
-                ),
-                owner = owner
-              ))
+          if (owner) {
+            Future.successful(
+              Ok(
+                views.html.dashboard.assessments(
+                  model = AssessmentsVM(
+                    assessmentsWithLinks = link.assessments
+                      .sortBy(_.currentFromDate.getOrElse(LocalDate.of(2017, 4, 7)))(
```

Comment:
We should factor this date out. Maybe config?

Reply:
Good point. I will do this in a separate commit